### ISSUE TITLE
error in DateTimeInput is not handled correctly? 

### ISSUE THREAD

**Issue by Voxis** [arfedulov/semantic-ui-calendar-react#22]
```jsx
<DateTimeInput
  placeholder="Date Time"
  className="example-calendar-input"
  name="dateTime"
  value={this.state.dateTime}
  iconPosition="left"
  error={true} // added
  onChange={this.handleChange} />
```

so I added the `error={true}` here but the error highlighting is not rendered. 
not sure what caused this. but the class `error` is correctly set in the html. 

------ 
Edit...

maybe it is just me, I added bunch of important to the css
```css
.ui.input.error > input {
  background-color: #fff6f6 !important;
  border-color: #e0b4b4 !important;
  color: #9f3a38 !important;
  -webkit-box-shadow: none !important;
  box-shadow: none !important;
}
```

and it worked.

**Comment by arfedulov** [comment]
This rule ``.ui.input.error > input`` gets overriden because this module uses *semantic-ui-react*'s ``Input`` instead of ``Form.Input``. There are some consequences of changing one to another so it needs to be tested properly. I'll probably do it on this weekend.

**Comment by arfedulov** [comment]
It's fixed in abda4fb9059dc68ec09da3072e3e1d86463d58b1.
